poco/services/environment_utils.py

```diff
@@ -17,8 +17,8 @@ def set_variable(key, value):
     @staticmethod
     def set_poco_uid_and_gid():
         if os.name == "posix":
-            EnvironmentUtils.set_variable("POCO_UID", str(os.getuid()))
```

Comment:
That was necessary, because if you not want to use your own user if run (for example if run as root, it isn't recommended the root privilege)